frontend/src/sagas/event.js

```diff
@@ -118,6 +113,13 @@ function* events() {
     }
 }
 
-export default function* eventSaga() {
-    yield fork(events)
+export default function eventSaga() {
+    return new Promise((resolve, reject) => {
+        const protocol = location.protocol === "https:" ? "wss:" : "ws:";
+        const url = `${protocol}//localhost:3000/ws`;
+        const p = connectTo(url);
```

Comment:
makes it easier to debug when things is not chained.. once it's proven stable it should be refactored into a more compact format for sure :)